kie-issues#58-Add boolean logic examples to the DMN FEEL handbook #58 (#5)
kiegroup/dmn-feel-handbook: source/index.html.md

You can use the `in expression` to check if a given value is matched by a specified range.

> ## boolean expression(and, or)

Reviewer: ```suggestion
## Three-valued logic (and, or)
```

> Examples

```FEEL,commented
true and true   //➔ true
true and false and null   //➔ false
true and null and true   //➔ null
true or false or null   //➔ true
false or false   //➔ false
false or null or false  //➔ null
true or false and false   //➔ true
(true or false) and false   //➔ false

> ```

Reviewer: ```suggestion
```

```

maintain a new-line to avoid syntax highlight confusion

> (true or false) and false   //➔ false
> ```

Reviewer: ```suggestion
(true or false) and false   //➔ false
```

```

maintain a new line to avoid syntax highlight confusion

> ```

Reviewer: I can't get github to properly handle a github suggestion, one reason more after ` ``` ` do maintain an empty newline, to avoid syntax highlight confusion

> You can use `and` and `or expressions` as the classic and (&&), and or (||) operators in other languages, with the addition that the result can evaluate to null.

Reviewer: ```suggestion
FEEL supports three-valued logic (3VL) semantic. You can use `and` and `or` as the classic and (`&&`), and or (`||`) operators in other languages, with the addition that the result can evaluate to null.
```
